src/core/PatternClip.cpp

```diff
@@ -60,6 +60,7 @@ void PatternClip::saveSettings(QDomDocument& doc, QDomElement& element)
 		element.setAttribute( "pos", startPosition() );
 	}
 	element.setAttribute( "len", length() );
+	element.setAttribute("off", startTimeOffset());
```

Comment:
I'd prefer we use the full description of the attribute, `offset`, instead of `off`. It can be confused with `off` (meaning turn off).

Granted this is in a file that no one should edit manually, but there's no reason we can't be descriptive.

Reply:
I just copied this line over from the SampleClip.cpp. I thought it would be good to be consistent across files here. Guess, I will just change both now before it breaks any files after a release.
And I indeed had to edit project files manually some time ago (sample paths weren't matching anymore).

Reply:
See newest review.

Reply:
> Guess, I will just change both now before it breaks any files after a release.

There are at least three choices:
- Leave everything as-is
- Change `off` to `offset` only in `PatternClip`: inconsistent with `SampleClip`, but no compatibility issues
- Change both in `PatternClip` and `SampleClip`: offsets in new files will be ignored in old versions, the converse can be handled by either trying with `off` as a fallback or using `DataFile::upgrade`